### PR TITLE
Assignment eight

### PR DESCRIPTION
**Pull request by Lauraxcav**
Record time& no crying.

### CODE REVIEW

**Comment by Lauraxcav** [comment]
Merging

File: lauracavanaugh-app/.idea/libraries/Maven__com_googlecode_json_simple_json_simple_1_1_1.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
+  <library name="Maven: com.googlecode.json-simple:json-simple:1.1.1">
+    <CLASSES>
```
Comment (smarks) [inline lauracavanaugh-app/.idea/libraries/Maven__com_googlecode_json_simple_json_simple_1_1_1.xml]: don't include any files under the ./idea directory

File: lauracavanaugh-app/src/main/java/edu/lauracavanaugh/advancedjava/week2/ServiceFactory.java [hunk 2]
```diff
@@ -26,7 +25,11 @@ public static StockService getService() {
         // In here we can decide which service to return
         // but for now, we're returning a BasicStockService
         //return new BasicStockService();
-        return SERVICE;
+        return new BasicStockService();
+    }
+
+    public static StockService getStockServiceInstance() {
+        return new AdapterClass(new AlphaVantageService());
```
Comment (smarks) [inline lauracavanaugh-app/src/main/java/edu/lauracavanaugh/advancedjava/week2/ServiceFactory.java]: what is an AlphaVantageService service ?

File: lauracavanaugh-app/src/main/java/com/origamisoftware/teach/advanced/model/StockQuery.java [hunk 3]
```diff
@@ -23,8 +23,8 @@
 
     // add a query interval here; used to be in StockService, but kind of makes sense in here too
     public enum IntervalEnum {
-        //MINUTE(Calendar.MINUTE),
-        //HOUR(Calendar.HOUR_OF_DAY),
+        MINUTE(Calendar.MINUTE),
+        HOUR(Calendar.HOUR_OF_DAY),
         DAY(Calendar.DAY_OF_YEAR),
```
Comment (smarks) [inline lauracavanaugh-app/src/main/java/com/origamisoftware/teach/advanced/model/StockQuery.java]: this enum is duplicated. only need it one place. make an enum class.

**Pull request closed without merging**